server/poll/transform.go

```diff
@@ -64,7 +64,7 @@ func (p *Poll) makeAdditionalText(numberOfVotes int) string {
 
 	lines := []string{"---"}
 	if len(settingsText) > 0 {
-		lines = append(lines, fmt.Sprintf("**Poll settings**: %s", strings.Join(settingsText, ", ")))
+		lines = append(lines, fmt.Sprintf("**Poll Settings**: %s", strings.Join(settingsText, ", ")))
```

Comment:
I changed this to be in line with the Readme. Hope you are fine with this @kaakaa.